dom-querying/dom-query-quiz-notes.md

```diff
@@ -0,0 +1,125 @@
+# dom-query-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Why do we log things to the console?
+  to test scripts
+- What is a "model"?
+
```

Comment:
Missing a quiz question

Reply:
Right you are

Reply:
![image](https://github.com/kaitlin-lovrich/c-0823-code-solutions/assets/69818777/56929fe0-a7e2-4906-8abf-039cac2f4a98)